apps/kampus/features/url/generate-url.ts

```diff
@@ -0,0 +1,46 @@
+import { env } from "~/env";
+
+export function getCommentURL({
+  baseUrl,
+  postID,
+  commentID,
+}: {
+  baseUrl: string;
+  postID: string;
+  commentID: string;
+}) {
+  const postURL = getPostURL({ postID, baseUrl });
+  const commentUrl = `${postURL}#c_${commentID}`;
+  return commentUrl;
+}
+
+export function getPostURL({ postID, baseUrl }: { postID: string; baseUrl: string }) {
+  const postUrl = baseUrl + getPostLink(postID);
+  return postUrl;
+}
+
+export function getPostLink(postID: string) {
+  const postUrl = `/posts/${postID}`;
+  return postUrl;
+}
+
+export function getSitePostsLink(post: { site: string }) {
+  const postUrl = `/site/${post.site}`;
+  return postUrl;
+}
+
+type KampusProduct = "sozluk" | "pano" | "pasaport";
+
+export const generateBaseURL = (product: KampusProduct) => {
```

Comment:
let's rename this to `getBaseURL` and move it to it's own file: `get-base-url.ts`

Reply:
```suggestion
export const getKampusURL = (product: KampusProduct, path: string = "") => {
```

let's accept an optional second argument so that we can directly use this helper to generate the full url with one function call:

```ts
const postURL = getKampusURL("pano", `/posts/${postID}`)
```